ee/modules/160-multitenancy-manager/hooks/apis/deckhouse.io/v1alpha1/projecttemplate_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2023 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ProjectTemplateSpec struct {
+	// 	List of ServiceAccounts, Groups and Users to provide access to the created project (isolated environment).
+	Subjects []AuthorizationRule `json:"subjects,omitempty" yaml:"subjects,omitempty"`
```

Comment:
Why not `AuthorizationRules` then?

```suggestion
	AuthorizationRules []AuthorizationRule `json:"subjects,omitempty" yaml:"subjects,omitempty"`
```